test/specs/reference/tests/01.reference.js

```diff
@@ -263,6 +263,18 @@ exports.execute = function (options) {
                 expect(tuple.values).toBeDefined();
             });
 
+            it('tuple.deepCopyData should create a shallow copy of the tuple except for the data.', function() {
+                var newTuple = tuple.deepCopyData();
+
+                // The original and new Tuple should not be the same object, neither should the _data
+                expect(newTuple).not.toBe(tuple);
+                expect(newTuple._data).not.toBe(tuple._data);
```

Comment:
Right, but your next test should be to `expect(newTuple._data[i].toBe(tuple._data[i])` (in a loop) to test that your function actually did the shallow copy of the keys of `_data`.